### PR TITLE
Added observer set for network activity

### PR DESCRIPTION
**Pull request by skytan4**

### CODE REVIEW

File: LDSContent/Session.swift [hunk 1]
```diff
@@ -22,8 +22,17 @@
 
 import Foundation
 import Operations
+import Swiftification
 
 class Session: NSObject {
+    
+    let contentNetworkActivityObservers = ObserverSet<NetworkActivity>()
```
Comment (hiltonc) [inline LDSContent/Session.swift]: It was networkActivityObservers in the other lib, any reason to diverge?

Comment (skytan4) [inline LDSContent/Session.swift]: No, sorry no good reason, just forgot to change it to match. I have some other changes coming though so expect to see that changed.

File: LDSContent/ContentController.swift [hunk 2]
```diff
@@ -101,6 +108,16 @@ public class ContentController {
             } catch {}
         }
     }
+    
+    private func notifyNetworkActivity(notification: NetworkActivity) {
+        switch notification {
```
Comment (nickmshelley) [inline LDSContent/ContentController.swift]: This can be changed to `networkActivityObservers.notify(notification)`

**Comment by nickmshelley** [comment]
.

File: LDSContent/ContentController.swift [hunk 3]
```diff
@@ -110,12 +110,7 @@ public class ContentController {
     }
     
     private func notifyNetworkActivity(notification: NetworkActivity) {
```
Comment (stephanheilner) [inline LDSContent/ContentController.swift]: Why is this var named "notification"? Seems like it should be called "networkActivity"

**Comment by nickmshelley** [comment]
Stephan had a comment. Looks like it's collapsed as outdated for some reason.

**Comment by nickmshelley** [comment]
👍

**Pull request merged**